hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -224,21 +235,179 @@
               - Read rx data oversampled value and ensure it is same as driven value
             '''
       milestone: V2
-      tests: []
+      tests: ["i2c_rx_oversample"]
     }
     {
-      name: rw_loopback
+      name: host_rw_loopback
       desc: '''
-            Test write data, read loopback, then compare.
+            Host and Target mode: do write data, read loopback, and compare.
 
             Stimulus:
-              - Drive i2c host to write data to the device then read loopback
+              - Drive DUT/Agent to write data to the Agent/DUT then read loopback
 
             Checking:
               - Ensure read data is matched with write data
             '''
       milestone: V2
-      tests: []
+      tests: ["i2c_rw_loopback"]
+    }
+
+    //-----------------------------------------------
+    // Tests for I2C DUT in TARGET mode
+    //-----------------------------------------------
+    {
+      name: target_smoke
+      desc: '''
+            Smoke test in which random (rd/wr) transactions are
+            sent to the DUT and received asynchronously with scoreboard checks.
+
+            Stimulus:
+              - Configure DUT/Agent to Target/Host mode respectively
+              - Enable DUT target
+              - Clear/Enable interrupt (if needed)
+              - Randomize I2C timing in TIMING 0-4 registers and other parameters such as TL agent delays
+              - Randomize address and data for read/write transactions sent to the DUT by the agent
```

Comment:
How would randomizing an address work in this case? DUT will accept transactions only if the address matches one of its own.

Reply:
@igk8 test can generate two randomized addresses which are used for i) config target_id register ii) transaction addresses sent by i2c_agent (host mode)